js/apps/admin-ui/src/realm-settings/TokensTab.tsx

```diff
@@ -179,8 +179,8 @@ export const RealmSettingsTokensTab = ({
                       id="oAuthDevicePollingInterval"
                       value={field.value}
                       min={0}
-                      onPlus={() => field.onChange(field.value || 0 + 1)}
-                      onMinus={() => field.onChange(field.value || 0 - 1)}
+                      onPlus={() => field.onChange(Number(field?.value) + 1)}
+                      onMinus={() => field.onChange((Number(field?.value) > 0) ? Number(field?.value) - 1 : 0)}
```

Comment:
This should satisfy the linter.

```suggestion
                      onMinus={() =>
                        field.onChange(
                          Number(field?.value) > 0
                            ? Number(field?.value) - 1
                            : 0
                        )
                      }
```